junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTestExtension.java

```diff
@@ -86,8 +85,9 @@ private ParameterizedTestNameFormatter createNameFormatter(Method templateMethod
 	}
 
 	private Object[] consumedArguments(Object[] arguments, Method templateMethod) {
-		int parametersCount = templateMethod.getParameterCount();
-		return arguments.length > parametersCount ? Arrays.copyOf(arguments, parametersCount) : arguments;
+		/*int parametersCount = templateMethod.getParameterCount();
+		return arguments.length > parametersCount ? Arrays.copyOf(arguments, parametersCount) : arguments;*/
```

Comment:
This should still be in place unless one of the parameters is an `ArgumentsAccessor` or annotated with `@AggregateWith`, i.e. unless `isAggregate(parameter)` is true for any method parameter.

Reply:
Should i make isAggregate static, or move it to an Util class then (or something else)? Instantiating a specific extension at that point seemed wrong semantically which is why i temporarily commented it out until i find a better way. Unless you disagree.

Reply:
You can make it `static` for now. I plan to go over the code in an IDE before we merge it and will potentially refactor a few things.